### ISSUE TITLE
SyntaxError: Unexpected token export

### ISSUE THREAD

**Issue by seemsindie** [Greentube/localize-router#9]
Hello guys, this is my module:

```ts
import { NgModule } from '@angular/core';
import { UniversalModule } from 'angular2-universal';
import { FormsModule } from '@angular/forms';
import { AppComponent } from './index';
import { AdminLoginComponent } from './admin-login/admin-login.component';
import { HomeComponent } from './home/home.component';
import { DashboardComponent } from './dashboard/dashboard.component';
import { ApartmentComponent } from './apartment/apartment.component';
import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
import { RouterModule } from '@angular/router';
import { routes } from './app.routes';

import { HttpModule, Http } from '@angular/http';
import { TranslateModule, TranslateLoader, TranslateStaticLoader, TranslateService } from 'ng2-translate';
import { LocalizeRouterModule, LocalizeParser, ManualParserLoader } from 'localize-router/localize-router';

/**
 * Top-level NgModule "container"
 */
@NgModule({
  /** Root App Component */
  bootstrap: [ AppComponent ],
  /** Our Components */
  declarations: [ AppComponent, AdminLoginComponent, HomeComponent, DashboardComponent, ApartmentComponent, PageNotFoundComponent ],
  imports: [
    /**
     * NOTE: Needs to be your first import (!)
     * BrowserModule, HttpModule, and JsonpModule are included
     */
    HttpModule,
    UniversalModule,
    FormsModule,
    /**
     * using routes
     */
    TranslateModule.forRoot({
      provide: TranslateLoader,
      useFactory: (http: Http) => new TranslateStaticLoader(http, '/assets/i18n', '.json'),
      deps: [Http] 
    }),
    LocalizeRouterModule.forRoot(routes, {
    provide: LocalizeParser,
        useFactory: (translate, http) => 
            new ManualParserLoader(translate, ['en','rs'], 'ROUTES'),
        deps: [TranslateService, Http]
    }), 
    RouterModule.forRoot(routes)
  ]
})
export class AppModule {

}

```

And when i run `ng serve` i am getting this error:

```
/home/ivan/Projects/Private/ApartmaniWeb/node_modules/localize-router/localize-router.js:1
(function (exports, require, module, __filename, __dirname) { export * from './src/localize-router.module';
                                                              ^^^^^^
SyntaxError: Unexpected token export
    at Object.exports.runInThisContext (vm.js:78:16)
    at Module._compile (module.js:543:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/ivan/Projects/Private/ApartmaniWeb/dist/server.bundle.js:692:18)
    at __webpack_require__ (/home/ivan/Projects/Private/ApartmaniWeb/dist/server.bundle.js:21:30)
```

Any idea what am i doing wrong?

**Comment by meeroslav** [comment]
@seemsindie 
Hi Ivan,
This looks like Typescript transpiler inconsistency error. `export` comes from ES6 format and your transpiler is probably set to common.js. It would help if you could provide more information:
- Which version of CLI are you using?
- What are your `tsconfig.json` settings?

**Comment by seemsindie** [comment]
@meeroslav 
My `tsconfig.json` contains:

```json
{
  "compilerOptions": {
    "declaration": false,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "lib": ["es6", "dom"],
    "mapRoot": "./",
    "module": "es6",
    "moduleResolution": "node",
    "outDir": "../dist/out-tsc",
    "sourceMap": true,
    "target": "es5",
    "typeRoots": [
      "../node_modules/@types"
    ],
    "types": [ "node" ]
  }
}
```

And i am using `universal-cli` https://github.com/devCrossNet/angular-cli
Version: 
universal-cli: 1.0.0-alpha.universal.1
angular-cli: 1.0.0-beta.19-3

**Comment by meeroslav** [comment]
I will look into it as soon as possible. Sorry for the delay.

**Comment by meeroslav** [comment]
Hi, can you compare your version with [https://github.com/meeroslav/universal-localize-example](https://github.com/meeroslav/universal-localize-example)? This is pretty rudimental project built with `universal-cli`.

Also I hope it's just a typo - project built with universal-cli should be served with `ung serve` not `ng serve`.

**Comment by phatpham9** [comment]
Anyone can help!? I have an error with `import { Observable } from 'rxjs/Observable'; SyntaxError: Unexpected token import`
![localize-router error](https://cloud.githubusercontent.com/assets/3795964/21093165/f69fae26-c082-11e6-850e-454b24d61427.png)

**Comment by Chuvisco88** [comment]
@phatpham9 I remember I had the same problem some time back. Did you add rxjs to your vendors?

**Comment by phatpham9** [comment]
@Chuvisco88 I've tried `import 'rxjs/Observable';` and now `SyntaxError: Unexpected token export`

**Comment by Wefeedthetroll** [comment]
Same issue trying to implement localize-router with angular universal ( universal start, not universal-cli ). I've tried to import rxjs but no luck.

**Comment by phatpham9** [comment]
Still waiting helps

**Comment by meeroslav** [comment]
Hi, can you provide minimal repo?

**Comment by kitsh** [comment]
@phatpham9 
I was having a very similar error as yours above. I was creating a repo to post here by mashing some stuff from preboot2 angular stub with this and the stub was updated to use webpack2.  I didn't have the error, I ported the configs from this back over to my main project
https://github.com/kitsh/Stub-ang2-webpack-localize-router
I think the problem had something to do with webpack 1 linking, I'm still not sure but I don't have more time to research.  I'll probably delete that repo at some point, because it a trash mashup I don't want another trash repo in my history... so grab it.
